PrebidMobile/PrebidMobileCore/src/main/java/org/prebid/mobile/core/TargetingParams.java

```diff
@@ -111,6 +113,47 @@ static void fetchLocationUpdates(Context appContext) {
     //endregion
 
     //region Public APIs
+    static final String consentStringKey = "Prebid_GDPR_consent_strings";
+
```

Comment:
Why write to Shared preference?? and fetch from it each time a request is made?? This might add to the latency in making the ad call each time.
Can this be handled as just instance variable or a combination of shared preference and instance variable.(or may be i am completely wrong)